src/models/vac.ts

```diff
@@ -9,14 +9,14 @@ import * as sddInformations from "../static/SDD.mapping.json";
  */
 export const genVaccinationRecord = (vaccinationRecord: VaccinationRecord, expiryDateTime: string, issuerName: string, reference: string): VaccineRecord[] => {
 
-  /* assign primary series of vaccination records by filtering `vaccinationDateTime` is before `effectiveDate` */
+  /* assign primary series of vaccination records by filtering `vaccinationDateTime` is before `effectiveDate` or without `effectiveDate` */
   const primaryVaccinations: Vaccination[] = (vaccinationRecord.vaccinations).filter((vaccination) =>
-    isBefore(parseISO(vaccination.vaccinationDateTime), parseISO(vaccinationRecord.effectiveDate))
+    vaccinationRecord.effectiveDate? isBefore(parseISO(vaccination.vaccinationDateTime), parseISO(vaccinationRecord.effectiveDate)): true
```

Comment:
I'm not sure if this logic will have any issue? Because if there is no `effectiveDate`, every vaccination will be counted as primary dose?

Reply:
yes. it's will be like this in ticket https://www.pivotaltracker.com/n/projects/2464296/stories/181533359. I'm already talk with darren about this, this should be consider as primary series if without have effective_date.